### PR TITLE
Update Maintainer JSON

### PR DESCRIPTION
**Pull request by Stargator**
@fwip @kabiir 

Would you like to be maintainers for this repo?

You both have done a terrific job and done a lot more than I could hope for. **Thank you**

There are no requirements in terms of amount of time spent or work done. Just whenever you can.

Plus, you can approve and then merge in PRs.

Just don't push directly to master any changes. We want anything big or small to be a PR.

### CODE REVIEW

**Comment by fwip** [comment]
Sure. :)

**Comment by Stargator** [comment]
@fwip Great! I forwarded that to @kytrinyx.

Let me know if you have any questions.

**Comment by kabiir** [comment]
@Stargator definitely!, currently I'm working on adding another new exercise called markdown it's about refactoring your code.

**Comment by kytrinyx** [comment]
Excellent! 🎉 I've invited you both to the team.

**Comment by Stargator** [comment]
@kabiir @fwip Let me know if there's anything in this PR you want changed. You don't have to have emails listed if you don't want or your email is otherwise listed on your profile.

**Comment by kabiir** [comment]
@Stargator for now, looks good to me. 👍

File: config/maintainers.json [hunk 1]
```diff
@@ -10,6 +10,36 @@
       "link_url": null,
       "name": null,
       "show_on_website": false
+    },
+    {
+      "alumnus": false,
```
Comment (kytrinyx) [inline config/maintainers.json]: There's been some back-and-forth about the "alumnus" key, but the default now is to either make it 'null' or remove it entirely for those who are not alumni.

https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md#alumnus

File: config/maintainers.json [hunk 2]
```diff
@@ -10,6 +10,36 @@
...
+      "avatar_url": "https://avatars0.githubusercontent.com/u/7527155?v=4&s=460",
+      "bio": null,
+      "github_username": "Stargator",
+      "link_text": null,
+      "link_url": null,
+      "name": null,
+      "show_on_website": false
```
Comment (kytrinyx) [inline config/maintainers.json]: Consider showing yourself on the website. You'd end up on the v2 site (not the v1) which is not live yet. Here's the page: https://v2.exercism.io/team

Comment (kabiir) [reply config/maintainers.json]: I'm guessing the website pulls bio from the github profile. If so, why is there a bio field?

Comment (kytrinyx) [reply config/maintainers.json]: Yepp, it does pull the bio if the bio is empty, but often people want to have a different bio depending on the language track, that says more about how they got into the language or what kind of work they focus on for the track.

File: config/maintainers.json (hunk 1 above)
Comment (Stargator) [reply config/maintainers.json]: Will remove it

**Comment by Stargator** [comment]
@fwip Any feedback? Just want to make sure I'm not missing anything from you before I merge.

**Comment by fwip** [comment]
Looks good to me!

**Comment by Stargator** [comment]
@fwip great!

**Pull request merged**